aws-datastore/src/test/java/com/amplifyframework/datastore/syncengine/PersistentMutationOutboxTest.java

```diff
@@ -691,13 +694,13 @@ public void nextItemForModelIdReturnsFirstEnqueued() throws DataStoreException {
         BlogOwner originalJoe = BlogOwner.builder()
             .name("Joe Swanson")
             .build();
-        PendingMutation<BlogOwner> firstMutation = PendingMutation.update(originalJoe, BlogOwner.class);
+        PendingMutation<BlogOwner> firstMutation = PendingMutation.update(originalJoe, BlogOwner.class, null);
         storage.save(originalJoe, converter.toRecord(firstMutation));
 
         BlogOwner updatedJoe = originalJoe.copyOfBuilder()
             .name("Joe Swanson, MD. (He finished med school, I guess?)")
             .build();
-        PendingMutation<BlogOwner> secondMutation = PendingMutation.update(updatedJoe, BlogOwner.class);
+        PendingMutation<BlogOwner> secondMutation = PendingMutation.update(updatedJoe, BlogOwner.class, null);
```

Comment:
This is a lot of null passing. Maybe we need to support this signature sans predicate?